x-pack/plugins/security_solution/server/lib/detection_engine/rule_response_actions/utils.test.ts

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getErrorProcessAlerts, getIsolateAlerts, getProcessAlerts } from './utils';
+import type { AlertWithAgent } from './types';
+
+const alert = {
```

Comment:
I suggest you not use a singleton mock across all of your tests and instead create a `getter` function to return unique objects per test

Reply:
same comment for other variables below